src/main/java/bio/terra/service/dataset/flight/inheritsteward/AdjustStewardMembersStep.java

```diff
@@ -41,7 +41,7 @@ private void addRemoveStewardMembers(FlightContext context, boolean inheritStewa
     for (var snapshotId : snapshotIds) {
       for (var email : custodians) {
         api.addRemoveMember(
-            userReq, IamResourceType.DATASNAPSHOT, snapshotId, IamRole.CUSTODIAN, email);
+            userReq, IamResourceType.DATASNAPSHOT, snapshotId, IamRole.STEWARD, email);
```

Comment:
We're setting this role on the snapshot, so it needs to be the steward role, not custodian.

Reply:
Whoops